Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageLayoutOptionsResolver.swift

```diff
@@ -36,8 +36,9 @@ open class _ChatMessageLayoutOptionsResolver<ExtraData: ExtraDataTypes> {
             messageIndexPath: indexPath,
             messages: messages
         )
-
-        var options: ChatMessageLayoutOptions = [
+        
+        // Do not show bubble if the message is to be rendered as large emoji
+        var options: ChatMessageLayoutOptions = message.shouldRenderAsJumbomoji ? [] : [
```

Comment:
nit: as soon as the bubble is no longer a default option, I'd handle it in a plain `if` as we do for other options
```swift
var options: ChatMessageLayoutOptions = [] 

if !message.shouldRenderAsJumbomoji {
   options.insert(.bubble)
}
```

Reply:
Makes sense 👍🏻  Changed the code to match this style